### ISSUE TITLE
Win 10 Pro v. 10.0.19041.84 Listening but [not supported]

### ISSUE THREAD

**Issue by vvambo** [stascorp/rdpwrap#1104]
Hiho,

I tried to install (1.6.2) via chocolatey, but that failed repeatedly, so I grabbed the files from the choco temp folder.

I manually started the install.bat, and tested with RDPCheck (could open 2 connections I think, didnt try to login 2 users) and checked RDPConf:  Wrapper 1.5.0.0 Installed, Service Running v. 10.0.19041.84, Listener Listening but [not supported]

I read up that this could be due to an old .ini and I went to download it from https://raw.githubusercontent.com/stascorp/rdpwrap/master/res/rdpwrap.ini
I stopped the RemoteDesktopService via 'services'. Replaced the  'rdwrap.ini'  in "C:/Program/RDP Wrapper"  (I noticed I couldnt directly save the file there due to rights) and then started the service again. 

RDPConf is giving me the same information, and if I try to login via RDP, I am asked to log off with my local user (even if I login to another user)

**Comment by mahoromax** [comment]
I read further into this, after I got confused by the many recent issues and followed along.

For others reading this:
This main repo is not really getting updates, there are other devs pushing updates to the .ini file.

I went with this one, the author was active in the issues here recently:
https://github.com/affinityv/INI-RDPWRAP/blob/master/rdpwrap.ini

And this works for me and solved my problems.

**Comment by vvambo** [comment]
I read further into this, after I got confused by the many recent issues and followed along.

For others reading this:
This main repo is not really getting updates, there are other devs pushing updates to the .ini file.

I went with this one, the author was active in the issues here recently:
https://github.com/affinityv/INI-RDPWRAP/blob/master/rdpwrap.ini

And this works for me and solved my problems.
